### DISCUSSION TITLE
problem in google chrome with this claim

### DISCUSSION THREAD

**Discussion by 04ichim** [ruffle-rs/ruffle#2387] [Q&A]
Hello you,
Today I bought frills for chrome connections and I get a lot of bugs and it's not a game where you can hear Flashplayer.
Maybe you can help me so that I can still play tomorrow.
Thank you very much.

![image](https://user-images.githubusercontent.com/77268642/104163950-7df27000-53f7-11eb-9c48-296038b2bfbc.png)

**Comment by Poopooracoocoo** [comment]
ActionScript 3 is a work in progress. Take a look at some of the documentation like the readme and the wiki :)
